maat-orchestration/src/test/java/uk/gov/justice/laa/crime/orchestration/service/IncomeEvidenceServiceTest.java

```diff
@@ -78,5 +80,45 @@ void givenValidWorkflowRequestAndRepOrder_whenUpdateEvidenceIsInvoked_thenApiSer
         verify(maatCourtDataApiService).updateFinancialAssessment(any(MaatApiUpdateAssessment.class));
         verify(incomeEvidenceMapper).maatApiAssessmentResponseToApplicationDTO(any(MaatApiAssessmentResponse.class),
                 any(ApplicationDTO.class));
-    }}
+    }
+
+    @Test
+    void givenAInitAssessmentIsInProgress_whenMangeIncomeEvidenceIsInvoked_thenCreateEvidenceShouldNotCalled() {
```

Comment:
See previous comment, these tests can be removed if we get rid of the `manageIncomeEvidence` method.